src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Processors/GeneralizeProcessor.cs

```diff
@@ -36,9 +42,9 @@ public ProcessResult Process(ElementNode node, ProcessContext context = null, Di
                         return result;
                     }
                 }
-                catch (InvalidOperationException ex)
+                catch (Exception ex)
```

Comment:
do we have a better place to handle this kind of errors?

Reply:
After offline discuss, we can inherit more exceptions from AnonymizerProcessingException and move this exception handling outside the function if necessary.